src/router.js

```diff
@@ -16,22 +16,50 @@ const routes = [
 		path: '/',
 		component: Home,
 	},
+	{
+		path: '/_/print/:collectiveSlugPart-:collectiveId(\\d+)',
+		component: CollectivePrintView,
+		props: (route) => route.params,
+	},
 	{
 		path: '/_/print/:collective',
 		component: CollectivePrintView,
 		props: (route) => route.params,
 	},
+	{
+		path: '/p/:token/print/:collectiveSlugPart-:collectiveId(\\d+)',
+		component: CollectivePrintView,
+		props: (route) => route.params,
+	},
 	{
 		path: '/p/:token/print/:collective',
 		component: CollectivePrintView,
 		props: (route) => route.params,
 	},
+	{
+		path: '/p/:token/:collectiveSlugPart-:collectiveId(\\d+)',
+		component: CollectiveView,
+		props: (route) => route.params,
+		children: [
+			{ path: 'page-:pageId(\\d+)-:pageSlug' },
```

Comment:
I think it's confusing to use `<slug>-<id>` for the collective URL part and `page-<id>-<slug>` for the page URL part. Why not use `<slug>-<id>` for the page as well?

Reply:
This was added as a prefix for some edge cases when generated slug/old file path contains digits and dash at the beginning